cloudformation.yaml

```diff
@@ -225,15 +259,32 @@ Resources:
                  "Next": "MigrateShards",
                  "Retry": [ {
                    "ErrorEquals": [ "States.ALL" ],
-                   "IntervalSeconds": 10,
-                   "MaxAttempts": 12
+                   "IntervalSeconds": 30,
+                   "MaxAttempts": 20,
+                   "BackoffRate": 1.0
                   } ]
                },
                "MigrateShards": {
                   "Type": "Task",
                   "Resource": "${MigrateShardsArn}",
-                  "End": true
-               }
+                  "Next": "ShardMigrationCheck"
+               },
+               "ShardMigrationCheck": {
+                  "Type": "Task",
+                  "Resource": "${ShardMigrationCheckArn}",
+                  "Next": "RemoveNode",
+                  "Retry": [ {
+                    "ErrorEquals": [ "States.ALL" ],
+                    "IntervalSeconds": 120,
+                    "MaxAttempts": 60,
+                    "BackoffRate": 1.0
+                   } ]
+               },
+               "RemoveNode": {
+                 "Type": "Task",
+                 "Resource": "${RemoveNodeArn}",
+                 "End": true
```

Comment:
All of these tasks will need a catch block, which triggers a separate lambda to notify us about an error (perhaps via Anghammarad), but I'll add that in a separate PR.